### PR TITLE
SMHI weather service

### PR DESCRIPTION
**Pull request by helto4real**
Docs for new Swedish weather service component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** https://github.com/home-assistant/home-assistant/pull/16717

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Took the liberty to fix some minor grammar & spelling errors.
Overall the docs look great, thanks!

✅ Approved. Can be merged as soon as the parent PR get merged in.

**Comment by helto4real** [comment]
Thanks @frenck for that, English is not my first language.

**Comment by frenck** [comment]
@helto4real That is ok! We are here to help 😄

File: source/_components/smhi.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+description: "Instructions on how to integrate SMHI forecasts within Home Assistant."
+date: 2018-09-23 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+featured: true
+logo: smhi.png
+ha_category: Hub
+ha_release: 0.80
```
Comment (MartinHjelmare) [inline source/_components/smhi.markdown]: 0.81

File: source/_components/smhi.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+To add SMHI weather forecast to your installation, go to the Integrations page inside the config panel and add a location by providing the longitude and latitude of your location.
+
+If the location is configured in Home Assistant, it will be selected as the default location. After that, you can add additional locations.
+
+<p class='note warning'>
+You can only add locations through the integrations page, not in configuration files.
+</p>
+
+<p class='note warning'>
+Only location close to sweden can be added. See [SMHI.se area](http://opendata.smhi.se/apidocs/metfcst/geographic_area.html) for more details what locations are supported.
```
Comment (MartinHjelmare) [inline source/_components/smhi.markdown]: "Only locations close to Sweden can be added."

File: source/_components/weather.smhi.markdown [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+description: "Instructions on how to integrate SMHI forecasts within Homeassistant."
+date: 2018-09-23 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+featured: true
+logo: smhi.png
+ha_category: Weather
+ha_release: 0.80
```
Comment (MartinHjelmare) [inline source/_components/weather.smhi.markdown]: 0.81

File: source/_components/weather.smhi.markdown [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+ha_iot_class: "Cloud Polling"
+---
+
+The `smhi` platform uses the [SMHI.se](https://www.smhi.se/) web service as a source for meteorological data for your location.
+
+## {% linkable_title Configuration %}
```
Comment (MartinHjelmare) [inline source/_components/weather.smhi.markdown:19]: Don't duplicate this information. Just refer the user to the component page.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**